GolfExample_TCAPI/assessmenttemplate.html

```diff
@@ -31,12 +32,19 @@
             }
         );
 
+        //Each go at the assessment is a new attempt at the assessment; there's no bookmarking within the assessment
         var stmt = {
-            verb: "attempted",
+            verb: {
+                id: "http://adlnet.gov/expapi/verbs/attempted",
+                display: {
+                    "en-US" : "attempted"
+                }
+            },
             object: testObj,
-            context: GolfExample.getContext(
-                GolfExample.CourseActivity.id
-            )
+            context: GolfExample.getContext(GolfExample.CourseActivity.id, true),
+            result: {
+                duration: "PT0H0M0S"

```

Comment:
No. I'd go so far to say that including the H and M is bad practice on my part. Please do fix!